script.js

```diff
@@ -1,4 +1,176 @@
+      input +
+      ". <br><br>" +
+      "You lose! Bummer. <br><br>" +
+      "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game!";
+  } //else
+  //var myOutputValue = "Please input the following : scissors , paper , rock";
+  console.log("myOutputValue for else");
+  console.log(myOutputValue);
+  //var randomChoice = generateRandomChoice(input);
+  //console.log("program choice");
+  //console.log(randomChoice);
+
+  // reversed choices scenarios
+  //reversed choice draw
+  if (input == randomChoiceReversed) {
+    myOutputValue =
+      "The computer chose " +
+      randomChoiceReversed +
+      ". <br>" +
+      "You chose " +
+      input +
+      ". <br><br>" +
+      "It's a draw! <br><br>" +
+      "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game! ";
+  }
+
+  // reversed win scenarios
+  // input = scissors, win randomchoice = paper
+  if (
+    (input == "reversed scissors" &&
+      randomChoiceReversed == REVERSED_CHOICE_ROCK) ||
+    // input paper, win randomchoice = rock
+    (input == "reversed paper" &&
+      randomChoiceReversed == REVERSED_CHOICE_SCISSORS) ||
+    // input rock , win randomchoice = scissors
+    (input == "reversed rock" && randomChoiceReversed == REVERSED_CHOICE_PAPER)
+  ) {
+    myOutputValue =
+      "The computer chose " +
+      randomChoiceReversed +
+      ". <br>" +
+      "You chose " +
+      input +
+      ". <br><br>" +
+      "Congratulations! It's a win! <br><br>" +
+      "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game!";
+  }
+  // reversed lose scenarios
+  if (
+    (input == "reversed scissors" &&
+      randomChoiceReversed == REVERSED_CHOICE_PAPER) ||
+    (input == "reversed paper" &&
+      randomChoiceReversed == REVERSED_CHOICE_ROCK) ||
+    (input == "reversed rock" &&
+      randomChoiceReversed == REVERSED_CHOICE_SCISSORS)
+  ) {
+    myOutputValue =
+      "The computer chose " +
+      randomChoiceReversed +
+      ". <br>" +
+      "You chose " +
+      input +
+      ". <br><br>" +
+      "You lose! Bummer. <br><br>" +
+      "Now you can type scissors, paper or rock to play another round! <br> You can also type Reversed Scissors, Reversed Paper, Reversed Rock to try the Reversed Game!";
+  }
   return myOutputValue;
 };
+
+// Define Choices
+var CHOICE_SCISSORS = "scissors";
+var CHOICE_PAPER = "paper";
+var CHOICE_ROCK = "rock";
+
+//generate random integer between scissor, paper and stone
+var generateRandomInteger = function (max) {
+  var randomDecimal = Math.random() * max;
+  var randomInteger = Math.floor(randomDecimal) + 1;
+  console.log(randomInteger);
+  return randomInteger;
+};
+
+var generateRandomChoice = function () {
```

Comment:
this should be at the top as a good practice to put the global variables and functions above the main function for readability